keyboards/atreus/keymaps/ridingqwerty/atreus.c

```diff
@@ -0,0 +1,23 @@
+/* Copyright 2018 Jack Humbert
```

Comment:
You shouldn't need this file in your keymap folder

Reply:
This can definitely go.